guides/common/attributes-satellite.adoc

```diff
@@ -157,11 +157,7 @@
 :RepoRHEL8ServerSatelliteUtils: satellite-utils-{RepoSatelliteVersion}-for-rhel-8-x86_64-rpms
 
 // RHEL 7 Satellite repos
-:RepoRHEL7ServerSatelliteMaintenanceProjectVersion: rhel-7-server-satellite-maintenance-{RepoSatelliteVersion}-rpms
-:RepoRHEL7ServerSatelliteServerProjectVersion: rhel-7-server-satellite-{RepoSatelliteVersion}-rpms
-:RepoRHEL7ServerSatelliteServerProjectVersionPrevious: rhel-7-server-satellite-{ProjectVersionPrevious}-rpms
 :RepoRHEL7ServerSatelliteToolsProjectVersion: rhel-7-server-satellite-client-6-rpms
```

Comment:
Looks like this attributes is only used once as value for another attribute. Out of scope but could be simplified:

````
$ rg -i RepoRHEL7ServerSatelliteToolsProjectVersion
guides/common/attributes-satellite.adoc
163::RepoRHEL7ServerSatelliteToolsProjectVersion: rhel-7-server-satellite-client-6-rpms
169::project-client-RHEL7-url: {RepoRHEL7ServerSatelliteToolsProjectVersion}
````

Reply:
I debated that, but I find it worrying we don't have `project-client-RHEL9-url` either so I decided to leave it out of scope. There's also a chance we'll just drop RHEL 7 in Satellite, but I don't know what the plan for that is right now.